mne_qt_browser/_pg_figure.py

```diff
@@ -1368,15 +1366,18 @@ def _q_font(point_size, bold=False):
 class EventLine(InfiniteLine):
     """Displays Events inside Trace-Plot"""
 
-    def __init__(self, pos, label, color):
+    def __init__(self, mne, pos, label, color):
         super().__init__(pos, pen=color, movable=False,
                          label=str(label), labelOpts={'position': 0.98,
                                                       'color': color,
                                                       'anchors': [(0, 0.5),
                                                                   (0, 0.5)]})
+        self.mne = mne
```

Comment:
You sure this should become part of the public API?

Reply:
EventLine shouldn't really used by the users. All other Qt-Objects also have `self.mne`, I just added it for consistency here in the same way. I think this is something originating from mne-python too where `mne` is also a public attribute of the mpl-browser.

Reply:
Ok!